cutscene_engine/cutscene_scripts/DialogueStarter.gd

```diff
@@ -0,0 +1,41 @@
+# Initializes the map's pawns and emits signals so the
+# Game node can start encounters
+# Processes in pause mode
+extends Node
+class_name DialogueStarter
+
+onready var dialogue_box = $DialogueInterface/DialogueBox
+#onready var trigger_box_setter = $TriggerBoxSetter
+
+var trigger_counter = 0
```

Comment:
This counter looks like it's never incremented past 1. Maybe it should be a bool

Reply:
its there in-case players can revisit the NPC again with extra dialogue. Just a potential though. If this isn't needed per request of the designers and/or director, I can make it a bool instead